tool/loadgen/main.go

```diff
@@ -53,32 +53,45 @@ func main() {
 	if cAdvisorURL != "" && containers == "" {
 		panic("flag -containers is required when -cadvisor is provided")
 	}
-	if strings.Contains(containers, ",") {
-		panic("not implemented: only one container name supported")
-	}
-	containerName := containers
 
 	log.Printf("Target is %q", targetURL)
 
 	waitUntilReady(targetURL, maxWait)
 	warmUp(targetURL, rps, warmupDuration)
 
-	stats := Stats{}
+	statsMap := make(map[string]Stats)
 	if cAdvisorURL != "" {
-		stats.Before = containerStats(cAdvisorURL, containerName)
+		for _, containerName := range strings.Split(containers, ",") {
+			imageName := strings.Split(containerName, "-")[0]
+
+			statsMap[imageName] = Stats{
+				Before: containerStats(cAdvisorURL, containerName),
```

Comment:
Did you consider changing the implementation of `containerStats` to fetch all data with a single request?

Reply:
Yes I did, but I elected to choose this approach over spending time to figure out how to make cAdvisor work